docs/diagrams/StorageClassDiagram.puml

```diff
@@ -18,7 +18,7 @@ package "LibTask Storage" #F4F6F6{
 Class "<<interface>>\nLibTaskStorage" as LibTaskStorage
 Class JsonLibTaskStorage
 Class JsonSerializableLibTask
-Class JsonAdaptedPerson
+Class JsonAdaptedPatron
 Class JsonAdaptedTag
```

Comment:
Goodjob with updating the StorageClassDiagram to reflect patron instead of person 👍